bigtable/tests/unit/test_table.py

```diff
@@ -627,6 +627,33 @@ def test_mutate_rows(self):
 
         self.assertEqual(result, expected_result)
 
+    def test_read_row_custom_retry(self):
+        from google.api_core import retry
+        from google.api_core.retry import if_exception_type
+        from google.cloud.bigtable import row_data
+        from google.cloud.bigtable.table import _BigtableRetryableError
+
+        credentials = _make_credentials()
+        client = self._make_client(project="project-id", credentials=credentials)
+        instance = client.instance(instance_id=self.INSTANCE_ID)
+        table = self._make_one(self.TABLE_ID, instance)
+        table.read_rows = mock.Mock(return_value=[])
+
+        custom_retry = retry.Retry(
+            predicate=if_exception_type(_BigtableRetryableError),
+            initial=1.0,
+            maximum=15.0,
+            multiplier=2.0,
+            deadline=120.0,  # 2 minutes
+        )
+        table.read_row(self.ROW_KEY, retry=custom_retry)
+        args, kwargs = table.read_rows.call_args
+        self.assertEqual(kwargs["retry"], custom_retry)
+
+        table.read_row(self.ROW_KEY)
+        args, kwargs = table.read_rows.call_args
+        self.assertEqual(kwargs["retry"], row_data.DEFAULT_RETRY_READ_ROWS)
+
```

Comment:
We generally prefer to call the function-under-test once per testcase.  Can you please split out this last call into a separate `test_read_row_default_retry` testcase?  E.g.:

```python
    def test_read_row_default_retry(self):
        from google.cloud.bigtable import row_data

        credentials = _make_credentials()
        client = self._make_client(project="project-id", credentials=credentials)
        instance = client.instance(instance_id=self.INSTANCE_ID)
        table = self._make_one(self.TABLE_ID, instance)
        table.read_rows = mock.Mock(return_value=[])

        table.read_row(self.ROW_KEY)
        args, kwargs = table.read_rows.call_args
        self.assertEqual(kwargs["retry"], row_data.DEFAULT_RETRY_READ_ROWS)
```

Reply:
Done, and noted for the future, thanks. Some extra blank lines have been added in `noxfile.py` as per PEP8 style guide.